app/mailers/order_assignment_mailer.rb

```diff
@@ -0,0 +1,19 @@
+class OrderAssignmentMailer < BaseMailer
+
+  def notify_assigned_user(order_details)
+    @order_details = Array(order_details)
+    return if @order_details.blank?
+
+    @user = @order_details.first.assigned_user
+    return if @user.blank?
```

Comment:
How would this happen? I could see if the job was delayed a while and the order detail had gotten unassigned in the meantime.